src/app/legacy/components/Promo/image.jsx

```diff
@@ -60,9 +60,9 @@ const createSizes = (useLargeImages, isProgrammeImage) => {
 
 const Image = props => {
   const { children, src, useLargeImages, ...rest } = props;
-  const isWebPSupported = WEBP_ORIGIN_CODES.some(originCode =>
-    src.includes(originCode),
-  );
+  const isWebPSupported = WEBP_ORIGIN_CODES.some(originCode => {
+    return src.includes(originCode);
+  });
```

Comment:
I think we can/should remove this completely

Reply:
removed